build/release/pre_cloudbuild.yaml

```diff
@@ -13,16 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 steps:
+
+  #
+  # Clean out the working dir, so we can replace it.
+  #
+  - name: ubuntu
+    script: rm *
+
   #
   # Shallow clone of Agones/main branch from the main repository
   #
   - name: gcr.io/cloud-builders/git
-    dir: /workspace
+    id: clone
     args:
       - clone
       - --depth=1
       - --branch=main
       - https://github.com/googleforgames/agones.git
+      - .
```

Comment:
./agones works for me

Reply:
Not 100% sure what you mean here, can you expand?

Reply:
I am receiving an error related to `.` at [this link](https://gist.github.com/Kalaiselvi84/7b4724b8ab4c8b06d65ab8ce46f7643d) and another error related to `script: rm *` at [this link](https://gist.github.com/Kalaiselvi84/c543a5e1d4643fcb4017dc538ad26f7c)

Reply:
Are you running the entire pull request, or just the changes in this file?

Reply:
Looking at your error message, you weren't also running with the changes to `release.mk` - is that correct?

Reply:
Yes, I have attempted to execute post_cloudbuild.yaml file directly, not via `post-build-release` target

Reply:
Both steps were completed successfully, and no issues were encountered with pre-build-release target

Reply:
Brilliant! Let's get this merged, and kickoff the release again.